### DISCUSSION TITLE
Prevent changing uploaded file names? + more

### DISCUSSION THREAD

**Discussion by kimdd89** [np-8/dash-uploader#129] [Q&A]
Hi @fohrloop, 
I have two quick questions, but doesn't seem like answers are available in the documentation.
I would very much like to get some insights.
By the way, this library is very helpful for what i am doing and really appreciate the effort went into the lib. Amazing!

Questions:

1. I have a extension that goes like this: *.s1p, *.s2p, *.s3p... where the numbers can be anything. I wonder if the filetype can handle such wildcard in the middle of the name?
2. When the file names are really long, it seem though the library automatically alias it with ~1, ~2 whatever. I wonder if this behavior can be suppressed?

Thanks,

**Comment by fohrloop** [comment]
Hi @kimdd89 ,

Unfortunately my answer to both is "I don't know". I haven't played around with dash-uploader for a while and I'm looking for someone else interested to maintain this. 

Perhaps your best option is to look at the source code and try to figure it out yourself.I hope you find the solutions!

Niko

**Reply by kimdd89** [reply]
Thanks for the reply. I hope you can find a someone who can maintain this library. I am too newbie in web dev to even offer my help.
Again thanks
